lib/rules/no-constructor-return.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview Rule to disallow returning value from constructor.
+ * @author Pig Fang <https://github.com/g-plane>
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        type: "suggestion",
+
+        docs: {
+            description: "disallow returning value from constructor",
+            category: "Best Practices",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/no-constructor-return"
+        },
+
+        schema: {},
+
+        fixable: "code",
```

Comment:
it could be `null`, since no fixer was implemented?

Reply:
Oh, it's a copy-paste error.